Sources/Networking/NetworkingError.swift

```diff
@@ -22,7 +22,7 @@ public enum NetworkingError: Error, LocalizedError, Equatable {
     case requestFailed(_ description: String)
     case urlError(_ code: Int, _ description: String? = nil)
     case encodingError(_ description: String)
-    
+    case knownError(data: Data, code: Int)
```

Comment:
I know I suggested that name, but it was mostly for example not to use it for sure, I suggest to find more standard name like clientSide error since it's between 400 to 500 or expectedErrorBetween400to500 ;-) what I mean it should show something more than known error since who knows what does known means!

Reply:
Makes fine sense. I suggested clientEntityError and pushed that. what do u think